pkg/commands/import/change_summarizer.go

```diff
@@ -0,0 +1,170 @@
+package _import
+
+import (
+	"fmt"
+	"strings"
+
+	kpack "github.com/pivotal/kpack/pkg/client/clientset/versioned"
+	k8serrors "k8s.io/apimachinery/pkg/api/errors"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	k8s "k8s.io/client-go/kubernetes"
+
+	importpkg "github.com/pivotal/build-service-cli/pkg/import"
+	buildk8s "github.com/pivotal/build-service-cli/pkg/k8s"
+	"github.com/pivotal/build-service-cli/pkg/lifecycle"
+)
+
+type changeSummarizer struct {
```

Comment:
Is this object necessary? Unlike pure object oriented languages it is necessary to create objects for each helper?